barcamp-theme/templates/base.html

```diff
@@ -61,7 +61,11 @@
 
 		    <p class="lead">Beaver BarCamp is an informal conference where everyone is encouraged to participate and the sessions are not predetermined. <a href="/pages/about.html"><em>Learn more</em></a></p>-->
 		    <h5>Event Sponsors</h5>
-        <p>Want to sponsor the event? Please contact info < at > beaverbarcamp.org</p>
+            <img class='sponsor' src='/theme/img/github.png' />
```

Comment:
Looks like they copied it from BBC11 and not from BBC14 (which seems to have only been visible to a few people). I just fixed that so the rest of the team can see it. We updated a lot of the information since BBC11. The one from BBC14 should be more accurate.